### PR TITLE
Fix rubocop errors

### PR DESCRIPTION
**Pull request by jessieay**
* These popped up after we upgraded the gem

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -18,12 +18,12 @@ Style/ArrayJoin:
...
   Enabled: false
 
-Naming/AsciiComments:
+Style/AsciiComments:
   Description: 'Use only ascii symbols in comments.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#english-comments'
   Enabled: false
 
-Style/AsciiIdentifiers:
+Naming/AsciiIdentifiers:
```
Comment (adarsh) [inline .rubocop.yml]: Whoops!

File: app/controllers/expenses_additional_sources_controller.rb [hunk 2]
```diff
@@ -6,11 +6,7 @@ class ExpensesAdditionalSourcesController < StandardStepsController
   def step_params
     super.tap do |params_hash|
       params_hash.each do |key, val|
-        params_hash[key] = if val == "true"
-                             true
-                           else
-                             false
-                           end
+        params_hash[key] = val == "true"
```
Comment (adarsh) [inline app/controllers/expenses_additional_sources_controller.rb]: I'm not sure I follow this...

File: app/helpers/mb_form_builder.rb [hunk 3]
```diff
@@ -179,11 +179,11 @@ def mb_checkbox(method, label_text, options = {})
...
-    if optional
-      optional_text = "<span class='form-card__optional'>(optional)</span>"
-    else
-      optional_text = ""
-    end
+    optional_text = if optional
+                      "<span class='form-card__optional'>(optional)</span>"
+                    else
+                      ""
+                    end
```
Comment (adarsh) [inline app/helpers/mb_form_builder.rb]: I personally prefer the previous version of this, but I suppose that's another PR.

File: spec/features/snap_application_with_maximum_info_spec.rb [hunk 4]
```diff
@@ -213,10 +213,10 @@ def upload_documents
   def add_document_photo(url)
     input = %(<input type="hidden" name="step[documents][]" value="#{url}">)
     page.execute_script(
-      <<~eos
+      <<~JAVASCRIPT
         document.querySelector('[data-documents-form]').
           insertAdjacentHTML('beforeend', '#{input}')
-      eos
+      JAVASCRIPT
```
Comment (adarsh) [inline spec/features/snap_application_with_maximum_info_spec.rb]: oh interesting... I prefer this, I believe.

File: app/controllers/expenses_additional_sources_controller.rb (hunk 2 above)
Comment (jessieay) [reply app/controllers/expenses_additional_sources_controller.rb]: this happened via rubocop's autocorrect. here it is in action:

```
params_hash = { test: "true" }
[10] pry(main)> params_hash.each do |key, val|
[10] pry(main)*   params_hash[key] = val == "true"
[10] pry(main)* end
=> {:test=>true}
[11] pry(main)> params_hash = { test: "blah" }
=> {:test=>"blah"}
[12] pry(main)> params_hash.each do |key, val|
[12] pry(main)*   params_hash[key] = val == "true"
[12] pry(main)* end
=> {:test=>false}
```

Comment (jessieay) [reply app/controllers/expenses_additional_sources_controller.rb]: agree that it's more confusing than the former!

File: app/helpers/mb_form_builder.rb (hunk 3 above)
Comment (jessieay) [reply app/helpers/mb_form_builder.rb]: 🤗

File: spec/features/snap_application_with_maximum_info_spec.rb (hunk 4 above)
Comment (jessieay) [reply spec/features/snap_application_with_maximum_info_spec.rb]: Yeah now rubocop wants 'meaningful' heredoc delimiters https://github.com/bbatsov/rubocop/pull/4467/files

(so anything other than EOS)

File: app/controllers/expenses_additional_sources_controller.rb (hunk 2 above)
Comment (jessieay) [reply app/controllers/expenses_additional_sources_controller.rb]: basically, it's evaluating `val == "true"` and setting the param key to whatever the result of that evaluation is (which is true or false)

**Pull request merged**